lib/bundler/env.rb

```diff
@@ -86,18 +86,40 @@ def self.git_version
       "not installed"
     end
 
+    def self.version_of(script)
+      return "not installed" unless Bundler.which(script)
+      %x"#{script} --version"
+    end
+
+    def self.chruby_version
+      return "not installed" unless Bundler.which("chruby-exec")
+      %x"chruby-exec -- chruby --version".lines.grep(/^chruby:/).first
+    end
+
     def self.environment
       out = []
 
       out << ["Bundler", Bundler::VERSION]
-      out << ["RubyGems", Gem::VERSION]
+      out << ["  Platforms", Gem.platforms.join(", ")]
       out << ["Ruby", ruby_version]
-      out << ["GEM_HOME", ENV["GEM_HOME"]] unless ENV["GEM_HOME"].nil? || ENV["GEM_HOME"].empty?
-      out << ["GEM_PATH", ENV["GEM_PATH"]] unless ENV["GEM_PATH"].nil? || ENV["GEM_PATH"].empty?
-      out << ["RVM", ENV["rvm_version"]] if ENV["rvm_version"]
-      out << ["Git", git_version]
-      out << ["Platform", Gem::Platform.local]
-      out << ["OpenSSL", OpenSSL::OPENSSL_VERSION] if defined?(OpenSSL::OPENSSL_VERSION)
+      out << ["  Full Path", Gem.ruby]
+      out << ["  Config Dir", Gem::ConfigFile::SYSTEM_CONFIG_PATH]
+      out << ["RubyGems", Gem::VERSION]
+      out << ["  Gem Home", ENV.fetch("GEM_HOME") { Gem.dir }]
```

Comment:
any reason we can use `Gem.paths.home` for this and `Gem.paths.path` for the next line?

Reply:
We can use those instead, I was just copying from the `gem env` command 👍

Reply:
we actually can't yet, since they're 1.8+. we'll transition over eventually